src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -33,6 +33,57 @@
 """ Space needed to reach the root of the literal block. """
 BASE_INDENT = " " * 12
 
+CATEGORIES_USED_ON_DESCRIPTION = sorted(
+    [
+        "angle per time",
+        "density",
+        "dimensionless",
+        "dynamic viscosity",
+        "emissivity",
+        "flow coefficient",
+        "heat transfer coefficient",
+        "length",
+        "mass flow rate",
+        "mass fraction",
+        "mass per mol",
+        "molar thermodynamic energy",
+        "molar volume",
+        "mole per mole",
+        "power",
+        "pressure",
+        "productivity index",
+        "specific heat capacity",
+        "standard volume per standard volume",
+        "standard volume per standard volume",
+        "standard volume per time",
+        "temperature",
+        "thermal conductivity",
+        "time",
+        "velocity",
+        "volume flow rate",
+        "volume fraction",
+        "volumetric thermal expansion",
+    ]
+)
+
+
+def generate_list_of_units(category: str) -> str:
+    """ Return an admonition with toggle to show the units available for a given category. """
+    from barril.units import UnitDatabase
+
+    unit_database = UnitDatabase.GetSingleton()
+    units = unit_database.GetCategoryInfo(category).valid_units_set
```

Comment:
Looking at the generated docs, I see that all units are being listed, while ALFAsim doesn't work with all of them (defined in `alfasim_gui/source/python/alfasim_gui/units.py` `RegisterUnits()`)

Example:
```length_units = ['m', 'ft', 'km', 'mm', 'in']```

Not sure what will happen if user uses one of the other units for length. If it works maybe there is no problem to list them all, could you check?

Reply:
hmm so this is weird, I thought that all configurations on barril were being made on alfasim_core, and I moved that file to alfasim-SDK.
I didn't know these configurations were spread across the application =/
So we need to move the  `alfasim_gui/units.py` to alfasim-SDK as well.

Reply:
Commit fd3668b move the content from alfasim_gui/units to alfasim_sdk